src/pymor/parameters/base.py

```diff
@@ -380,7 +380,7 @@ def strip_parameter(self, mu):
         assert all(getattr(mu.get(k, None), 'shape', None) == v for k, v in self.parameter_type.items())
         return Parameter({k: mu[k] for k in self.parameter_type})
 
-    def build_parameter_type(self, *args, **kwargs):
+    def build_parameter_type(self, *args, provides=None, **kwargs):
```

Comment:
This is invalid syntax in Python 2 and I would be surprised if 3to2 can handle that ...